src/longitudinal-and-multi-site-studies.md

```diff
@@ -62,6 +63,19 @@ A guide for using macros can be found at
     }
 ) }}
 
+`sub-control01_sessions.tsv` content:
+
+```Text
+session_id	acq_time	systolic_blood_pressure
+ses-predrug	2009-06-15T13:45:30	120
```

Comment:
Should we use anonymized (pre-1925) times?

Reply:
I grabbed it from the sessions.tsv section: should we anonymize in both places?

Reply:
Ah, if we're reusing, I'm cool with maintaining consistency.